MVCDemos/Content/DemosStyles/Drawer/LeftOrRightPosition.css

```diff
@@ -20,6 +21,7 @@
 .panel-list {
     height: 400px;
     padding-top: 12px;
+    background-color: var(--dx-color-main-bg);
```

Comment:
Seem, you skipped change for '#content' selector in the line 51. Please, check it in both approaches.

Reply:
ok will check on this one